backend/timebank/src/main/kotlin/kookmin/software/capstone2023/timebank/application/service/inqui/InquiryService.kt

```diff
@@ -0,0 +1,138 @@
+package kookmin.software.capstone2023.timebank.application.service.inqui
+
+
+import kookmin.software.capstone2023.timebank.domain.model.Inquiry
+import kookmin.software.capstone2023.timebank.domain.model.InquiryStatus
+import kookmin.software.capstone2023.timebank.domain.model.User
+import kookmin.software.capstone2023.timebank.domain.repository.InquiryRepository
+import kookmin.software.capstone2023.timebank.domain.repository.UserJpaRepository
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import java.time.LocalDateTime
+
+@Service
+class InquiryService(
+        private val inquiryRepository: InquiryRepository,
+        private val userJpaRepository: UserJpaRepository
+) {
+
+    /**
+     * Dto 클래스 정의
+     */
+
+    data class InquiryDto(
+            val inquiryid: Long,
+            val title: String,
+            val content: String,
+            val replyContent: String?,
+            val inquiryDate: LocalDateTime,
+            val replyDate: LocalDateTime?,
+            val replyStatus: InquiryStatus,
+            val userId: Long
+    )
+
+    /**
+     * 질문생성Dto
+     */
+    data class InquiryCreateRequest(
+            val title: String,
+            val content: String,
+            val userId: Long,
+            val inquiryDate: LocalDateTime = LocalDateTime.now()
+    )
+
+    /**
+     * 답변 Dto
+     */
+    data class InquiryUpdateRequest(
+            val replyContent: String?,
+            val replyStatus: InquiryStatus?,
+            val replyDate: LocalDateTime? = LocalDateTime.now()
+    )
+
+    /**
+     * 서비스 메소드 정의
+     */
+
+    /**
+     * 문의 생성 service
+     */
+    @Transactional
+    fun createInquiry(request: InquiryCreateRequest): InquiryDto {
```

Comment:
```suggestion
    fun createInquiry(request: InquiryCreateRequest): Inquiry {
```

요건 필수사항은 아닌데 서비스는 다른 서비스에서도 호출할 수 있기 때문에
DTO로 매핑해서 내려주기보다는 Domain Model 자체를 내려주고 사용하는 곳에서 DTO로 매핑하는 방법도 있어요.

Reply:
참고하겠습니다!